lib/clevertap_plugin.dart

```diff
@@ -39,26 +40,31 @@ class CleverTapPlugin {
       cleverTapPushAmpPayloadReceivedHandler;
   late CleverTapPushClickedPayloadReceivedHandler
       cleverTapPushClickedPayloadReceivedHandler;
-  late CleverTapPushPermissionResponseReceivedHandler cleverTapPushPermissionResponseReceivedHandler;
+  late CleverTapPushPermissionResponseReceivedHandler
+      cleverTapPushPermissionResponseReceivedHandler;
   static List<CleverTapOnVariablesChangedHandler>
       cleverTapOnVariablesChangedHandlers = [];
-  static List<CleverTapOnValueChangedHandler>
-      cleverTapOnValueChangedHandlers = [];
+  static List<CleverTapOnValueChangedHandler> cleverTapOnValueChangedHandlers =
+      [];
 
-  static const MethodChannel _dartToNativeMethodChannel = const MethodChannel('clevertap_plugin/dart_to_native');
-  static const MethodChannel _nativeToDartMethodChannel = const MethodChannel('clevertap_plugin/native_to_dart');
+  static const MethodChannel _dartToNativeMethodChannel = const MethodChannel(
+      'clevertap_plugin/dart_to_native', const StandardMethodCodec());
+  static const MethodChannel _nativeToDartMethodChannel =
+      const MethodChannel('clevertap_plugin/native_to_dart');
 
   static final CleverTapPlugin _clevertapPlugin =
       new CleverTapPlugin._internal();
 
   factory CleverTapPlugin() => _clevertapPlugin;
 
   static const libName = 'Flutter';
+
   static const libVersion = 10901; // If the current version is X.X.X then pass as X0X0X
```

Comment:
@KambleSonam @shivamsharma2710 I think we should make the Flutter release v2.0.0 or `20000`